src/Umbraco.Web/Editors/MediaController.cs

```diff
@@ -278,6 +281,31 @@ public IEnumerable<MediaItemDisplay> GetByIds([FromUri]int[] ids)
         }
 
         /// <summary>
+        /// Searches media and returns a paged result
+        /// </summary>
+        /// <param name="query"></param>
+        /// <param name="pageNumber"></param>
+        /// <param name="pageSize"></param>
+        /// <param name="searchFrom"></param>
+        /// <returns></returns>
+        [HttpGet]
+        [FilterAllowedOutgoingMedia(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IMedia>>), "Items")]

```

Comment:
@Shazwazza well it is properties about the image like crops for getting a src for the thumbnail we show in the mediapicker grid. But I think maybe @madsrasmussen is better suited to answer that.

I don't know when it started using search but in 7.5.9 it's using it:
![image](https://cloud.githubusercontent.com/assets/706823/22920451/637f9490-f295-11e6-835c-dd66bf892971.png)